test/integration/app.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { program } from  '../../src/command/controller';
+
+describe('AppController', () => {
+    beforeEach(async () => {
+        program.exitOverride();
+        program.configureOutput({
+            writeOut: () => {},
+            writeErr: () => {}
+        });
+    });
+
```

Comment:
What are you testing exactly with 'expect.anything()`?

Reply:
Ideally I would like to avoid using a lot of mocks on application tests, but I ran out of time to experiment with other libraries such as pg-mem for a in-memory database, as I wanted to deliver the project on the date we agreed on.

At the moment those tests just run until they hit an error due to the lack of proper dependencies. Not really testing any output as I dont have independent implementation of such dependencies